src/server/api/routes/cvs.router.js

```diff
@@ -92,10 +92,7 @@ router.post('/', (req, res) => {
     .catch((error) => {
       console.log(error);
 
-      res
-        .status(400)
-        .send('Bad request')
-        .end();
+      res.status(400).send('Bad request').end();
```

Comment:
You shouldn't need to change this line.

Reply:
okey,.i have redone it,.,.the newest code is attached below,., Thanks Patrick